### PR TITLE
add healthecheck api endpoint

### PR DESCRIPTION
**Pull request by wuast94**
Hopefully i did all right. 

Checked functionality locally.

was quite hard to find out how to disable the hooks (ESlint checking failed)

### CODE REVIEW

**Comment by wuast94** [comment]
will close #326

File: apps/web/app/api/health/route.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { NextRequest, NextResponse } from "next/server";
+import fetch from "node-fetch";
+
+import serverConfig from "../../../../../packages/shared/config";
+
+const checkMeilisearchHealth = async () => {
```
Comment (MohamedBassem) [inline apps/web/app/api/health/route.ts]: I'd keep it simple and just not check meili's health in this endpoint.

**Review by MohamedBassem — CHANGES_REQUESTED** [review]
Hey @wuast94, I'm really sorry it took me that long to get to this PR. Thanks a lot for taking the time to implement it. I'd understand if you no longer have time to address the requested feedback :)

**Comment by wuast94** [comment]
> Hey @wuast94, I'm really sorry it took me that long to get to this PR. Thanks a lot for taking the time to implement it. I'd understand if you no longer have time to address the requested feedback :)

Didnt see it sry 😅 The healthcheck gets very simple then because its just the response if i didnt miss anything.

File: apps/web/app/api/health/route.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { NextRequest, NextResponse } from "next/server";
+
+export const GET = async (_req: NextRequest) => {
+  const webAppStatus = { status: "ok", message: "Web app is working" };
+
+  return NextResponse.json({
+    webApp: webAppStatus,
```
Comment (MohamedBassem) [inline apps/web/app/api/health/route.ts]: Do you mind just inlining the status in the response? Something like:
```suggestion
    status: "ok",
    message: "All good"
```

**Comment by MohamedBassem** [comment]
@wuast94 yeah, the healthcheck is mostly making sure that the web server is up and running.

**Comment by wuast94** [comment]
dont know why i left the variable in there but you are right on this, good catch 😊 maybe because im not a programmer xD

**Comment by MohamedBassem** [comment]
Thanks @wuast94!

**Pull request merged**